common/models/user.json

```diff
@@ -23,6 +23,7 @@
       "type": "string",
       "required": true
     },
+    "emailCaseSensitive": "boolean",
```

Comment:
`emailCaseSensitive` is a model setting, not a model property. Please remove this line.